cook-book/getting-started/README.md

```diff
@@ -0,0 +1,287 @@
+         failover_priority:
+@@ -53,3 +66,12 @@ all:
+           - storage-1-replica
+         roles:
+           - vshard-storage
++
++    replicaset_core_1:
++      hosts:
++        core-1:
++      vars:
++        replicaset_alias: core-1
++        roles:
++          - vshard-router
++          - failover-coordinator
+```
+
+### Configure instances
+
+To start and configure instance we can use
+[`configure_instances`](/doc/scenario.md#scenarios) pre-defined scenario.
+But it also contain steps we don't need (like `deliver_package` and `update_package`).
+`update_instance` step is used only with [multiversion approach](/doc/multiversion.md)
+, so it can be removed too.
+
+Use this `cartridge_scenario` in [playbook.configure-instances.yml](./playbook.configure-instances.yml):
+
+```yaml
+cartridge_scenario:
+  - configure_instance
+  - restart_instance
+  - wait_instance_started
+```
+
+Run new playbook:
+
+```bash
+ansible-playbook -i hosts.yml playbook.configure-instances.yml
+```
+
+Go to http://localhost:8182/admin/cluster/dashboard:
+
+![image](./images/instances-configured.png)
+
+Now we have two new unconfigured instances.
+
+### Accessing instances status and logs
+
+To check instance systemd service just connect to virtual machine and call
+`systemctl status`:
+
+```bash
+vagrant ssh vm1
+[vagrant@localhost ~]$ sudo systemctl status myapp@core-1
+● myapp@core-1.service - Tarantool Cartridge app myapp@core-1
+   Loaded: loaded (/etc/systemd/system/myapp@.service; enabled; vendor preset: disabled)
+   Active: active (running) since Fri 2021-04-02 17:33:14 UTC; 4min 33s ago
+  Process: 6413 ExecStartPre=/bin/sh -c mkdir -p /var/lib/tarantool/myapp.%i (code=exited, status=0/SUCCESS)
+ Main PID: 6414 (tarantool)
+   CGroup: /system.slice/system-myapp.slice/myapp@core-1.service
+           └─6414 tarantool init.lua: myapp@core-1
+```
+
+You can also use wildcard to get all `myapp` instance status:
+
+```bash
+sudo systemctl status myapp@*
+```
+
+Instances logs are available with `journalctl`.
+It can be helpful to pipe them to `less`:
+
+```bash
+sudo journalctl -u myapp@core-1 | less
+```
+
+### Configure topology
+
+When we sure that instances have started successfully it's time to join them
+to replicasets.
+
+We will use [`configure_topology`](/doc/scenario.md#scenarios) pre-defined scenario
+to manage replicasets.
+
+Specify `cartridge_scenario_name` in [playbook.configure-topology.yml](./playbook.configure-topology.yml):
+
+```yaml
+cartridge_scenario_name: configure_topology
+```
+
+Ansible allows to [limit](https://docs.ansible.com/ansible/latest/user_guide/intro_patterns.html)
+hosts configured by one playbook.
+It's convenient to manage one replicaset by specifying replicaset group.
+
+First, join new instance to `storage-1` replicaset:
+
+```bash
+ansible-playbook -i hosts.yml playbook.configure-topology.yml \
+    --limit replicaset_storage_1
+```
+
+You can check new topology on http://localhost:8182/admin/cluster/dashboard.
```

Comment:
What about to add image here too?

Reply:
There is image after adding one more instance.
But maybe it's Ok, I will ad one more image.